bin/crap

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
It saves you lots of hour of frustration with shringcrap. We can argue the name but imo that should be a script. However the important is the sync of version.